creating database

takashiandGH opened the pull request:
**what**
read meを書いた

**why**
データベース設計のため

takashiandGH committed:
read meつくった！！

naokishizuka commented:
タイトルやコメントは（自分とは関係ない人も含めて）複数の人が読む可能性が高いので、もうちょっとお硬い感じの方がいいですね。

naokishizuka commented on README.md:
```diff
@@ -0,0 +1,68 @@
+###groups
+|name|type|null|
+|:--:|:--:|:--:|
+|group_name|strings|false|
```
group.group_nameとなると冗長なので
nameでいきましょう。

naokishizuka commented on README.md:
```diff
@@ -0,0 +1,68 @@
+|name|type|null|
+|:--:|:--:|:--:|
+|body|text||
+|image|strings||
```
stringだね

naokishizuka commented on README.md:
```diff
@@ -0,0 +1,68 @@
+|image|strings||
+|group_id|integer|false|
+|user_id|integer|false|
+|group_id|integer|false|
```
group idが２つ。。。

naokishizuka commented on README.md:
```diff
@@ -0,0 +1,68 @@
+
+##アソシエーション
+###users
+######has_many groups　　
```
#######って書くとh6要素になっちゃうから
li要素とかにしましょう

takashiandGH committed:
前回のものから頂いたコメントをもとに修正した

ShuheiTsuji commented on README.md:
```diff
@@ -0,0 +1,67 @@
+* belongs_to group
+
+
+## Description
```
ここから下は消してあげましょう。

ShuheiTsuji commented on README.md:
```diff
@@ -0,0 +1,67 @@
+|group_id|integer|false|
+|user_id|integer|false|
+
+###group_users
```
テーブル名であれば多対多なのでgroups_usersテーブルの方が良いです。

ShuheiTsuji commented on README.md:
```diff
@@ -0,0 +1,67 @@
+###users
+* has_many groups :through, group_users
+* has_many messages
+
```
has_many :group_usersも書いておきましょう。

ShuheiTsuji commented on README.md:
```diff
@@ -0,0 +1,67 @@
+* has_many messages
+
+###groups
+* has_many users :through, group_users
```
has_many :group_usersも書いておきましょう。

ShuheiTsuji commented:
あとはindexをどの際に記述するか書いていただくと！😊

takashiandGH committed:
・コメントをもとに修正
・messagesとgroup_usersをrefarence型に変更
・messagesとgroup_usersにindex、foreign_keyについての記述を追加

takashiandGH committed:
微修正

ShuheiTsuji commented:
lgtmで！！😊
